### PR TITLE
Update basics

### PR DESCRIPTION
**Pull request by DaanVanHauwermeiren**
updated the basics

- [ ] do we need to cover macro's in this?
- [ ] only some things on strings. Do we need to add more?
- [x] will types and multiple dispatch be covered further?
- [x] same for structs?

### CODE REVIEW

File: basics.jmd [hunk 1]
```diff
@@ -17,7 +17,9 @@ Let's get started with the basics. Some mathematical operations,
 
 2 / 4       # standard division
 
-div(2, 4)   # Computes 2/4 truncated to an integer
```
[the remark this answers is not in the document]
Comment (MichielStock) [reply basics.jmd]: `div` is quite useful

**Comment by MichielStock** [comment]
I also like the exercises, but there will be far too many to do in an afternoon.

**Comment by MichielStock** [comment]
@DaanVanHauwermeiren: will you resolve the conflicts and merge?

**Comment by Beramos** [comment]
Any progress on this so I can make the final changes

**Comment by DaanVanHauwermeiren** [comment]
@Beramos: working on it atm!

File: basics.jmd (hunk 1 above)
[the remark this answers is not in the document]
Comment (DaanVanHauwermeiren) [reply basics.jmd]: added again in #4517628

[the remark this answers is not in the document]
Comment (DaanVanHauwermeiren) [reply basics.jmd]: added again in 3c665ed04fc7b1921648167b4f32094f117122eb

**Comment by DaanVanHauwermeiren** [comment]
It took me 1 hour to fix all the merge conflicts. **Please do not ever again push directly into master**. Thanks

**Comment by Beramos** [comment]
Thx, I never push directly into the master. Do you think that is the reason for the merge conflicts though?

**Pull request merged**